modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -525,6 +483,21 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 			scopeSiteObjectDefinition);
 	}
 
+	@Test
+	public void testObjectEntry1toMObjectUnrelatedModelsProviderImpl()
+		throws Exception {
+
+		_testSystemObjectEntry1toMObjectUnrelatedModels(
+			CompanyThreadLocal.getCompanyId());
+
+		Company company2 = CompanyTestUtil.addCompany();
+
+		_testSystemObjectEntry1toMObjectUnrelatedModels(
+			company2.getCompanyId());
+
```

Comment:
@murilostodolni can we rename it to `company` as we don't have another `company1` here?